provider/README.md

```diff
@@ -201,13 +201,17 @@ Schema: [`trips` schema][trips-schema]
 
 The trips API should allow querying trips with a combination of query parameters.
 
-* `device_id`
-* `vehicle_id`
 * `min_end_time`: filters for trips where `end_time` occurs at or after the given time
 * `max_end_time`: filters for trips where `end_time` occurs before the given time
+* `static_end_time`: An [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date for a static bucket of files representing either a Month, Day, or Hour. Timezone UTC. 
 
-When multiple query parameters are specified, they should all apply to the returned trips. For example, a request with `?min_end_time=1549800000000&max_end_time=1549886400000` should only return trips whose end time falls in the range `[1549800000000, 1549886400000)`.
+| Request range | format | expected output | 
+| ------------- | ------ | --------------- | 
+| Month         | YYYY-DD | That month's worth of trips, or if the month is not complete or the file does not exist , an error | 
+| Day           | YYYY-MM-DD | That day's worth of trips, or if the day is not complete or the file does not exist, an error | 
+| Hour          | YYYY-MM-DD:HH | That hour worth of trips, or if the hour is not complete or hte file does not exist, an error. The minimum value of the hour should be 00 and the max 23, as the midnight hour should be denotated as the next day's zero hour. | 
 
+The `min_end_time` and `max_end_time` endpoints shall not be used in combination with the `static_end_time` endpoints, rather, the the former is good for real-ish time uses cases, while the latter is best for reconstructing the historical record and backfilling. It is considered permissive to return an incorrect query parameter responses for the `min_end_time`  / `max_end_time` if the range contains timestamps older than one month. 
```

Comment:
Small nitpick: I think that "uses cases" should be "use cases" and that "permissive" should be "permissible".

I'm actually not sure what "It is considered permissive to return an incorrect query parameter responses for the `min_end_time`  / `max_end_time` if the range contains timestamps older than one month." means - what's an "incorrect query parameter response"? If you're referring to a response with a particular HTTP status code (maybe 400?), I think it would be helpful to directly state that status code here. Thanks!

Reply:
removed in new draft, thanks for the feedback @jrheard